app/controllers/claims/support/schools_controller.rb

```diff
@@ -6,4 +6,44 @@ def index
   def show
     @school = Claims::School.find(params.require(:id))
   end
+
+  def new
+    @school = Claims::School.new
+  end
+
+  def check
+    if school.valid? && !school.claims?
+      @school = school.decorate
+    else
+      school.errors.add(:urn, :taken) if school.claims?
+      render :new
+    end
+  end
+
+  def create
+    school.claims = true
```

Comment:
It doesn't seem to set it by default

Reply:
This is because the `school` association of a `GiasSchool` will return `School`, not a namespaced model.
This is important, as using `Claims::School` will not recognise if the school has already been added by Placements, causing it to error. When in actual fact all we want to do in that case is trigger the flag.